pkg/apis/kubeone/v1beta2/defaults.go

```diff
@@ -219,6 +220,14 @@ func SetDefaults_MachineController(obj *KubeOneCluster) {
 	}
 }
 
+func SetDefaults_OperatingSystemManager(obj *KubeOneCluster) {
+	if obj.OperatingSystemManager == nil {
+		obj.OperatingSystemManager = &OperatingSystemManagerConfig{
+			Deploy: true,
+		}
+	}
+}
```

Comment:
Should we do this only if MC is deployed? I guess it doesn't make much sense otherwise.

Reply:
Eventually, it will be able to work without machine-controller, in cases where someone wants to configure their machines, after provisioning them. But yes for now, maybe it makes sense to check for MC as well.